### ISSUE TITLE
[BUG] The tag ‘$:/tags/Macro’ affects the animation of the Tiddler.

### ISSUE THREAD

**Issue by LinHQ1999** [Jermolene/TiddlyWiki5#7415]
### Describe the bug

After editing a tiddler that contains the tag `$:/tags/Macro`, the animation effect cannot be seen normally.

### Expected behavior

The animation can be displayed normally.

### To Reproduce

1. Create a tiddler that tagged with `$:/tags/Macro`.
3. Confirm changes to this tiddler
4. The animation of the tiddler has disappeared.

### TiddlyWiki Configuration

- It can be reproduced on the official documentation.

**Comment by pmario** [comment]
I can confirm the behaviour. I think it has always been there. ... IMO starting with v5.1.5

I think it's not possible to change this behaviour. I think ... As soon as the tiddler tagged: $:/tags/Macro is saved, the story river will be redrawn, which interrupts the current animation.

I would consider this as an inconvenience.

**Comment by LinHQ1999** [comment]
> I can confirm the behaviour. I think it has always been there. ... IMO starting with v5.1.5
> 
> I think it's not easy to change this behaviour. I think ... As soon as the tiddler tagged: $:/tags/Macro is saved, the story river will be redrawn, which interrupts the current animation.
> 
> I would consider this as an inconvenience.

I agree, it‘s indeed just some kind of flaw. I only discovered this issue after going through several versions…
